include/boost/histogram/python/register_axis.hpp

```diff
@@ -53,29 +53,6 @@ struct options {
     bool growth() const { return static_cast<bool>(option & bh::axis::option::growth); }
 };
 
-/// Ensure metadata is valid.
-inline void validate_metadata(metadata_t v) {
```

Comment:
We need to add the kargs and get metadata from that, can be in a later PR, though. I can do that.

Reply:
It looks ok to me as it is right now. The test all work. Since the number of positional arguments is fixed for each axis now, there is no ambiguity anymore. I think we should leave it as is.

Reply:
I'll make another PR or issue, I really don't like allowing someone to pass metadata without a keyword. It's not clear what they mean. Not important for this PR.